fixups/FileRedirectionFixup/PathRedirection.h

```diff
@@ -30,8 +30,8 @@ struct path_redirect_info
     bool shouldReadonly = false;
 };
 
-path_redirect_info ShouldRedirect(const char* path, redirect_flags flags);
-path_redirect_info ShouldRedirect(const wchar_t* path, redirect_flags flags);
+path_redirect_info ShouldRedirect(const char* path, redirect_flags flags, DWORD inst=0);

```

Comment:
nit: add spaces on both sides of the = sign.

Reply:
Addressed in Commit 1bdc01a3